### PR TITLE
Atpesc2019

### PR DESCRIPTION
**Pull request by rhornung67**
Some minor changes related to the recent ATPESC tutorial

### CODE REVIEW

File: test/unit/cuda/CMakeLists.txt [hunk 1]
```diff
@@ -21,13 +21,13 @@ raja_add_test(
   NAME test-cuda-reduce-loc
   SOURCES test-reduce-loc.cpp)
 
-raja_add_test(
-  NAME test-cuda-reduce-tupleloc
-  SOURCES test-reduce-tupleloc.cpp)
+#raja_add_test(
```
Comment (davidbeckingsale) [inline test/unit/cuda/CMakeLists.txt]: Should these be commented?

Comment (rhornung67) [reply test/unit/cuda/CMakeLists.txt]: Shoot. Thanks for catching that. Change pushed with comments removed.

Comment (rchen20) [reply test/unit/cuda/CMakeLists.txt]: For now, yes they ought to commented out. Those tests are breaking on CUDA8+clang5 that Argonne is using. The tests compile on all of our builds, and pass at runtime.

Comment (rhornung67) [reply test/unit/cuda/CMakeLists.txt]: Since they pass all of our builds, they should not be commented out.

**Pull request merged**